contributing.qmd

```diff
@@ -2,63 +2,83 @@
 title: Get Involved
 ---
 
-We encourage contributions to this guide.
+## Introduction
+
+We encourage contributions to this guide. The guide's goal on cloud-optimized geospatial formats is to provide documentation on the best practices for the current state-of-the-art cloud-optimized formats. These formats are evolving, and so will the guide.
 
 ## Pre-requisites
 
 If you wish to preview the site locally, install [quarto](https://quarto.org/). You will also need to be familiar with [quarto markdown](https://quarto.org/docs/authoring/markdown-basics.html).
 
-## Core principles
+## Communication Channels
+
+Discussions can occur in [GitHub Discussions](https://github.com/developmentseed/cloud-optimized-geospatial-formats-guide/discussions) and issues can be raised at [GitHub Issues](https://github.com/developmentseed/cloud-optimized-geospatial-formats-guide/issues).
```

Comment:
In case contributors might not be familiar with Github, maybe add one sentence for when to create a discussion vs issue?